packages/utils/types/NodePath.ts

```diff
@@ -0,0 +1,76 @@
+export interface INode extends Object {
+	id: {
+		name: string;
+	};
+	arguments: string[];
+	body: INode[];
+	elements: INode[];
+	expression: {
+		left: {
+			computed: boolean,
+			object: INode,
+			property: INode,
+			type: string,
+		},
+		operator: string,
+		right: INode,
+		type: string,
+	};
+	filter: (p: (p: INode) => boolean) => INode[] | any;
+	find: (objectExpression: object, filterExpression?: object) => INode;
+	forEach: (p: (p: INode) => void) => INode[] | any;
+	get: (property: string) => INode;
+	remove: () => void;
```

Comment:
_ : void for all missing

Reply:
It's a promise, so _ doesn't go here. Please let's not spread this to normal functions